app/routes/tasks.js

```diff
@@ -5,11 +5,20 @@ const API_BASE_URL = ENV.BASE_API_URL;
 
 export default class TasksRoute extends Route {
   model = async () => {
-    const response = await fetch(`${API_BASE_URL}/tasks/self`, {
-      credentials: 'include',
-    });
-    const allTasks = await response.json();
-
-    return allTasks;
+    try {
+      const response = await fetch(`${API_BASE_URL}/tasks/self`, {
+        credentials: 'include',
+      });
+      const allTasks = await response.json();
+      if (!response.ok) {
+        if (response.status == 401) {
+          throw new Error('Please log in to continue');
```

Comment:
Where will this error show up? Shouldn't we use `alert` browser API?
Also we should directly redirect them to realdevsquad.com for logging in

Reply:
I have used `alert` inside catch block. Error will show up whenever control reaches that catch block

Reply:
Cool!